src/java/main/org/apache/zookeeper/ClientCnxnSocket.java

```diff
@@ -63,6 +65,12 @@
     protected long now;
     protected ClientCnxn.SendThread sendThread;
     protected LinkedBlockingDeque<Packet> outgoingQueue;
+    protected ZKClientConfig clientConfig;
+    /**
```

Comment:
It is a matter of taste tbh, but I usually prefer to add this kind of comment as an one-line just after constant. Like,
```
private static int CLIENT_MAX_PACKET_LENTH_DEFAULT = 4096 * 1024;   // 4 MB
```